text/0084-integer-shard-index.md

```diff
@@ -0,0 +1,581 @@
+# Integer shard index
+
+- RFC PR: https://github.com/tikv/rfcs/pull/0084
+- Tracking Issue: https://github.com/pingcap/tidb/issues/31040
+
+## Summary
+
+Hot index reduces the write scalability of the TiDB cluster when the written data is monotonically increasing. We find out a proposal to solve the problem that is using an expression index to scatter the hot index, the new expression index is called a shard index.
```

Comment:
What's the table schema and what's the insert workload?

Reply:
No matter what table schema, but it must has a index and the insert data for this index is monotonically increasing.